contrib/importers/roboform2pass.rb

```diff
@@ -33,7 +33,7 @@ def ask_required_info
 
   def save
     if valid?
-      IO.popen("pass insert -m -f '#{path}' > /dev/null", "w") do |pass_io|
```

Comment:
Instead of using `pw edit ...` it should write here directly to `"#{tempdir}#{path}"`, like this:
https://github.com/dashohoxha/pw/blob/master/contrib/importers/csv2pw.sh#L38-L39

For it to work, you need to create first a `tempdir`, like this:
https://github.com/dashohoxha/pw/blob/master/contrib/importers/csv2pw.sh#L22-L23

Finally, we need to import the content of `tempdir` to `pw` and erase `tempdir`:
https://github.com/dashohoxha/pw/blob/master/contrib/importers/csv2pw.sh#L48-L52

By the way, it also needs to get a `pw` archive as an argument:
https://github.com/dashohoxha/pw/blob/master/contrib/importers/csv2pw.sh#L19-L20